apps/www/src/actions/generate-api-key.ts

```diff
@@ -13,16 +13,22 @@ export async function generateAndSaveApiKey() {
 	const user = await getCurrentUser();
 	const userId = user?.id;
 
+	if (!process.env.UNKEY_ROOT_KEY) {
+		console.error("UNKEY_ROOT_KEY environment variable is not set.");
+		return {
+			success: false,
+			error: "UNKEY_ROOT_KEY environment variable is not set.",
+		};
+	}
+
 	const unkey = new Unkey({ rootKey: process.env.UNKEY_ROOT_KEY });
 
 	if (!userId) {
 		console.error("No user is currently logged in.");
 		return { success: false, error: "User not authenticated" };
 	}
 
-	const {
-		result: { key: apiKey },
-	} = await unkey.keys.create({
+	const { result } = await unkey.keys.create({
 		apiId: "api_3PkKmeLT2WeGsbRUVWH1YAZJK886",
```

Comment:
I think the problem is that this is @domeccleston's hardcoded apiID, @Codehagen needs to use their own from the unkey dashboard.